src/app/resources/resources-add.component.html

```diff
@@ -0,0 +1,93 @@
+<h2 i18n>New Resource</h2>
+<form [formGroup]="resourceForm" (ngSubmit)="onSubmit()" novalidate>
+  <mat-grid-list cols="3">
+    <mat-form-field>
+      <input matInput i18n-placeholder placeholder="Title"  formControlName="title" required>
+      <mat-error><planet-form-error-messages [control]="resourceForm.controls.title"></planet-form-error-messages></mat-error>
```

Comment:
right now its mat-error and planet-form-error-messages, can we make it just to be one that will have both features